components/bt/host/bluedroid/api/include/api/esp_gap_ble_api.h

```diff
@@ -17,2687 +17,3916 @@
+                                             - E.g. Devices that require user input for PIN codes (6 bytes), such as smart locks or car systems. */
+    ESP_BLE_SM_AUTHEN_REQ_MODE,         /*!< Authentication mode requirements of the local device, size 1 byte (unit 8).
+                                             - Scenario: Defines the local device's need for pairing, encryption, and whether to use MITM (Man-In-The-Middle) protection or secure connections.
+                                             - See `esp_ble_auth_req_t` for details. 
+                                             - Group define for `ESP_LE_AUTH_xxx`. 
+                                             - E.g. Devices with high security needs, like financial payment devices. */
+    ESP_BLE_SM_IOCAP_MODE,              /*!< Input/output capabilities of the local device in pairing, size 1 byte (unit 8).
+                                             - Scenario: Defines whether the device supports keyboard input, display output, or has no I/O capabilities, which affects the pairing method.
+                                             - See `esp_bt_io_cap_t` for details. 
+                                             - Group define for `ESP_BT_IO_CAP_xxx`. */
+    ESP_BLE_SM_SET_INIT_KEY,            /*!< Initiator key distribution during pairing, size 1 byte (unit 8).
+                                             - Scenario: Defines the types of keys distributed by the local device during pairing. */
+    ESP_BLE_SM_SET_RSP_KEY,             /*!< Responder key distribution during pairing, size 1 byte (unit 8).
+                                             - Scenario: Defines the types of keys distributed by the local device during pairing. */
+    ESP_BLE_SM_MAX_KEY_SIZE,            /*!< Maximum encryption key size supported by the device, size 1 byte (7-16).
+                                             - Scenario: Limits the maximum length of the encryption key that the local device can generate or accept. */
+    ESP_BLE_SM_MIN_KEY_SIZE,            /*!< Minimum encryption key size required from the peer device, size 1 byte (7-16).
+                                             - Scenario: Ensures that the paired device meets the minimum security requirements, preventing weak encryption. */
+    ESP_BLE_SM_SET_STATIC_PASSKEY,      /*!< Set a static passkey for pairing, size 6 byte (000000-999999).
+                                             - Scenario: Uses a fixed passkey for pairing instead of a randomly generated one. */
+    ESP_BLE_SM_CLEAR_STATIC_PASSKEY,    /*!< Clear previously set static passkey.
+                                             - Scenario: Removes the static passkey to allow pairing using the default random passkey or other security methods. */
+    ESP_BLE_SM_ONLY_ACCEPT_SPECIFIED_SEC_AUTH,  /*!< Accept only specified security authentication requirements, size 1 byte.
```

Comment:
The length of the value is 1 byte (uint8_t)
- Group define for `ESP_BLE_ONLY_ACCEPT_SPECIFIED_AUTH_xxx`.